### PR TITLE
fix hover text

### PR DESCRIPTION
**Pull request by britt6612**
Related Issue #418 

## Work Included 

* Added a box around the NavLinks in order to set a width so that when the text changed to bold it 
would expand which was causing the text to move

* Moved footerLinks above to an array to clean up some code then mapped through that array in the NavLinks

* Added Margin for horizontal to footer otherwise since I added in the width to the box it was being expanded in the footer example

### Mobile
Screen Shot 2020-02-27 at 12 38 54 PM
 
### Full Screen

Screen Shot 2020-02-27 at 12 38 19 PM

### CODE REVIEW

**Comment by britt6612** [comment]
@taysea thanks for catching that! After trying the different approaches I think this is something to be discussed with the team and designers about other approaches

**Comment by ericsoderberghp** [comment]
I'm concerned that the use of bold text as a hover indicator for Anchor tends to cause issues like this and that we should discuss whether that's the best style to use in the theme. Agreed on wanting a discussion. :)

**Comment by britt6612** [comment]
@ericsoderberghp I agree 100%. Especially if were trying to provide an out of the box solution for users to have. Setting a fixed width in a footer example is not something we would want. It was beneficial for me to try and explore different approaches for us to then go back to the design to discuss.

**Comment by britt6612** [comment]
> I'm concerned that the use of bold text as a hover indicator for Anchor tends to cause issues like this and that we should discuss whether that's the best style to use in the theme. Agreed on wanting a discussion. :)

@L0ZZI @gregfuruiye

File: aries-site/src/layouts/main/Footer.js [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
-import { NavLink } from 'aries-core';
+import { FooterLink } from 'aries-core';
 import { Box, Footer as GrommetFooter, ResponsiveContext, Text } from 'grommet';
 import { nameToPath } from '../../utils';
 
 export const Footer = () => {
   const size = useContext(ResponsiveContext);
   const year = new Date().getFullYear();
+
+  const footerLinks = [
```
Comment (taysea) [inline aries-site/src/layouts/main/Footer.js]: Should we rename this to be `externalFooterLinks` to be clear why `Feedback` isn't part of this?

File: aries-core/src/js/components/helpers/Anchors/FooterLink.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React, { forwardRef, useState } from 'react';
```
Comment (taysea) [inline aries-core/src/js/components/helpers/Anchors/FooterLink.js]: This is maybe a bigger discussion for after the PR, but I think we need to assess all of the anchor treatments that currently exist in aries-core and make sure that we want the design system to support all of them and then potentially revisit earlier discussions about having them live under a single component called `Anchor` with prop `type`. Thoughts? @halocline

File: aries-site/src/layouts/main/Footer.js (hunk 1 above)
Comment (britt6612) [reply aries-site/src/layouts/main/Footer.js]: yes good suggestion Ill change that

File: aries-core/src/js/components/helpers/Anchors/FooterLink.js (hunk 2 above)
Comment (halocline) [reply aries-core/src/js/components/helpers/Anchors/FooterLink.js]: @taysea Yes, I think the scope is larger than this PR. I added detail to this issue https://github.com/hpe-design/design-system/issues/308. Let's get on a call to talk through.

**Pull request merged**